README.md

```diff
@@ -307,69 +322,111 @@ standardVersion({
 })
 ```
 
-## Commit Message Convention, at a Glance
```

Comment:
I've removed this section – the README is pretty big and it seems like this is better handled by linking out to conventionalcommits.org.